.eslintrc

```diff
@@ -1,15 +1,15 @@
 {
   "parser": "babel-eslint",
-  "extends": "eslint:recommended",
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
```

Comment:
I wouldn't just turn on new lint rules, especially since no code has been changed to ensure the existing code doesn't fail.